ztp/source-crs/HardwareEventOperatorStatus.yaml

```diff
@@ -0,0 +1,26 @@
+# This CR verifies the installation/upgrade of the Baremetal Hardware Event Proxy Operator
+apiVersion: operators.coreos.com/v1
```

Comment:
why you need this ?

Reply:
but all the other kids in town have this...

Reply:
Typically the Subscription creates the operator automatically. This CR should not be needed (if it is we should dig into why...)

Reply:
deleted